From bba001a451ae79a983b3ecac40ff86c4121c83c8 Mon Sep 17 00:00:00 2001
From: Akshay S Dinesh <asdofindia@gmail.com>
Date: Wed, 21 Jun 2017 18:19:18 +0530
Subject: Whitelist Indian keyboard layouts. #101532

Indian keyboard layouts were moved into extras in #96418.
This was not necessary as it was indeed possible to type complex
characters of Indian scripts using these layouts.

Fixes: https://bugs.freedesktop.org/show_bug.cgi?id=101532

Signed-off-by: Akshay S Dinesh <asdofindia@gmail.com>

diff --git a/rules/base.extras.xml.in b/rules/base.extras.xml.in
index 530497c..d64d79b 100644
--- a/rules/base.extras.xml.in
+++ b/rules/base.extras.xml.in
@@ -994,372 +994,6 @@
         </variant>
       </variantList>
     </layout>
-    <layout>
-      <configItem>
-        <name>bd</name>
-        <!-- Keyboard indicator for Bangla layouts -->
-        <_shortDescription>bn</_shortDescription>
-        <_description>Bangla</_description>
-        <languageList>
-          <iso639Id>ben</iso639Id>
-          <!-- sat-Beng: http://www.ethnologue.com/language/sat -->
-          <iso639Id>sat</iso639Id>
-        </languageList>
-      </configItem>
-      <variantList>
-        <variant>
-          <configItem>
-            <name>probhat</name>
-            <_description>Bangla (Probhat)</_description>
-          </configItem>
-        </variant>
-      </variantList>
-    </layout>
-    <layout>
-      <configItem>
-        <name>in</name>
-        <!-- Keyboard indicator for Indian layouts -->
-        <_shortDescription>in</_shortDescription>
-        <_description>Indian</_description>
-      </configItem>
-      <variantList>
-        <variant>
-          <configItem>
-            <name>ben</name>
-            <!-- Keyboard indicator for Bangla layouts -->
-            <_shortDescription>bn</_shortDescription>
-            <_description>Bangla (India)</_description>
-            <languageList>
-              <iso639Id>ben</iso639Id>
-              <!-- sat-Beng: http://www.ethnologue.com/language/sat -->
-              <iso639Id>sat</iso639Id>
-            </languageList>
-          </configItem>
-        </variant>
-        <variant>
-          <configItem>
-            <name>ben_probhat</name>
-            <!-- Keyboard indicator for Bangla layouts -->
-            <_shortDescription>bn</_shortDescription>
-            <_description>Bangla (India, Probhat)</_description>
-            <languageList>
-              <iso639Id>ben</iso639Id>
-              <!-- sat-Beng: http://www.ethnologue.com/language/sat -->
-              <iso639Id>sat</iso639Id>
-            </languageList>
-          </configItem>
-        </variant>
-        <variant>
-          <configItem>
-            <name>ben_baishakhi</name>
-            <_description>Bangla (India, Baishakhi)</_description>
-            <languageList>
-              <iso639Id>ben</iso639Id>
-              <!-- sat-Beng: http://www.ethnologue.com/language/sat -->
-              <iso639Id>sat</iso639Id>
-            </languageList>
-          </configItem>
-        </variant>
-        <variant>
-          <configItem>
-            <name>ben_bornona</name>
-            <_description>Bangla (India, Bornona)</_description>
-            <languageList>
-              <iso639Id>ben</iso639Id>
-              <!-- sat-Beng: http://www.ethnologue.com/language/sat -->
-              <iso639Id>sat</iso639Id>
-            </languageList>
-          </configItem>
-        </variant>
-         <variant>
-          <configItem>
-            <name>ben_gitanjali</name>
-            <_description>Bangla (India, Uni Gitanjali)</_description>
-            <languageList>
-              <iso639Id>ben</iso639Id>
-              <!-- sat-Beng: http://www.ethnologue.com/language/sat -->
-              <iso639Id>sat</iso639Id>
-            </languageList>
-          </configItem>
-        </variant>
-        <variant>
-          <configItem>
-            <name>ben_inscript</name>
-            <_description>Bangla (India, Baishakhi Inscript)</_description>
-            <languageList>
-              <iso639Id>ben</iso639Id>
-              <!-- sat-Beng: http://www.ethnologue.com/language/sat -->
-              <iso639Id>sat</iso639Id>
-            </languageList>
-          </configItem>
-        </variant>
-        <variant>
-          <configItem>
-            <name>eeyek</name>
-            <_description>Manipuri (Eeyek)</_description>
-            <languageList>
-              <iso639Id>mni</iso639Id>
-            </languageList>
-          </configItem>
-        </variant>
-        <variant>
-          <configItem>
-            <name>guj</name>
-            <!-- Keyboard indicator for Gujarati layouts -->
-            <_shortDescription>gu</_shortDescription>
-            <_description>Gujarati</_description>
-            <languageList>
-              <iso639Id>guj</iso639Id>
-            </languageList>
-          </configItem>
-        </variant>
-        <variant>
-          <configItem>
-            <name>guru</name>
-            <!-- Keyboard indicator for Punjabi layouts -->
-            <_shortDescription>pa</_shortDescription>
-            <_description>Punjabi (Gurmukhi)</_description>
-            <languageList>
-              <iso639Id>pan</iso639Id>
-            </languageList>
-          </configItem>
-        </variant>
-        <variant>
-          <configItem>
-            <name>jhelum</name>
-            <!-- Keyboard indicator for Punjabi layouts -->
-            <_shortDescription>pa</_shortDescription>
-            <_description>Punjabi (Gurmukhi Jhelum)</_description>
-            <languageList>
-              <iso639Id>pan</iso639Id>
-            </languageList>
-          </configItem>
-        </variant>
-        <variant>
-          <configItem>
-            <name>kan</name>
-            <!-- Keyboard indicator for Kannada layouts -->
-            <_shortDescription>kn</_shortDescription>
-            <_description>Kannada</_description>
-            <languageList>
-              <iso639Id>kan</iso639Id>
-            </languageList>
-          </configItem>
-        </variant>
-        <variant>
-          <configItem>
-            <name>kan-kagapa</name>
-            <!-- Keyboard indicator for Kannada layouts -->
-            <_shortDescription>kn</_shortDescription>
-            <_description>Kannada (KaGaPa phonetic)</_description>
-            <languageList>
-              <iso639Id>kan</iso639Id>
-            </languageList>
-          </configItem>
-        </variant>
-        <variant>
-          <configItem>
-            <name>mal</name>
-            <!-- Keyboard indicator for Malayalam layouts -->
-            <_shortDescription>ml</_shortDescription>
-            <_description>Malayalam</_description>
-            <languageList>
-              <iso639Id>mal</iso639Id>
-            </languageList>
-          </configItem>
-        </variant>
-        <variant>
-          <configItem>
-            <name>mal_lalitha</name>
-            <!-- Keyboard indicator for Malayalam layouts -->
-            <_shortDescription>ml</_shortDescription>
-            <_description>Malayalam (Lalitha)</_description>
-            <languageList>
-              <iso639Id>mal</iso639Id>
-            </languageList>
-          </configItem>
-        </variant>
-        <variant>
-          <configItem>
-            <name>mal_enhanced</name>
-            <!-- Keyboard indicator for Malayalam layouts -->
-            <_shortDescription>ml</_shortDescription>
-            <_description>Malayalam (enhanced Inscript, with rupee)</_description>
-            <languageList>
-              <iso639Id>mal</iso639Id>
-            </languageList>
-          </configItem>
-         </variant>
-         <variant>
-           <configItem>
-            <name>ori</name>
-            <!-- Keyboard indicator for Oriya layouts -->
-            <_shortDescription>or</_shortDescription>
-            <_description>Oriya</_description>
-            <languageList>
-              <iso639Id>ori</iso639Id>
-              <!-- sat-Orya: http://www.ethnologue.com/language/sat -->
-              <iso639Id>sat</iso639Id>
-            </languageList>
-          </configItem>
-        </variant>
-        <variant>
-           <configItem>
-            <name>olck</name>
-            <!-- Keyboard indicator for Ol Chiki layouts -->
-            <_shortDescription>sat</_shortDescription>
-            <_description>Ol Chiki</_description>
-            <languageList>
-              <!-- sat-Olck: http://www.ethnologue.com/language/sat -->
-              <iso639Id>sat</iso639Id>
-            </languageList>
-          </configItem>
-        </variant>
-        <variant>
-          <configItem>
-            <name>tam_unicode</name>
-            <!-- Keyboard indicator for Tamil layouts -->
-            <_shortDescription>ta</_shortDescription>
-            <_description>Tamil (Unicode)</_description>
-            <languageList>
-              <iso639Id>tam</iso639Id>
-            </languageList>
-          </configItem>
-        </variant>
-        <variant>
-          <configItem>
-            <name>tam_keyboard_with_numerals</name>
-            <!-- Keyboard indicator for Tamil layouts -->
-            <_shortDescription>ta</_shortDescription>
-            <_description>Tamil (keyboard with numerals)</_description>
-            <languageList>
-              <iso639Id>tam</iso639Id>
-            </languageList>
-          </configItem>
-        </variant>
-        <variant>
-          <configItem>
-            <name>tam_TAB</name>
-            <!-- Keyboard indicator for Tamil layouts -->
-            <_shortDescription>ta</_shortDescription>
-            <_description>Tamil (TAB typewriter)</_description>
-            <languageList>
-              <iso639Id>tam</iso639Id>
-            </languageList>
-          </configItem>
-        </variant>
-        <variant>
-          <configItem>
-            <name>tam_TSCII</name>
-            <!-- Keyboard indicator for Tamil layouts -->
-            <_shortDescription>ta</_shortDescription>
-            <_description>Tamil (TSCII typewriter)</_description>
-            <languageList>
-              <iso639Id>tam</iso639Id>
-            </languageList>
-          </configItem>
-        </variant>
-        <variant>
-          <configItem>
-            <name>tam</name>
-            <!-- Keyboard indicator for Tamil layouts -->
-            <_shortDescription>ta</_shortDescription>
-            <_description>Tamil</_description>
-            <languageList>
-              <iso639Id>tam</iso639Id>
-            </languageList>
-          </configItem>
-        </variant>
-        <variant>
-          <configItem>
-            <name>tel</name>
-            <!-- Keyboard indicator for Telugu layouts -->
-            <_shortDescription>te</_shortDescription>
-            <_description>Telugu</_description>
-            <languageList>
-              <iso639Id>tel</iso639Id>
-            </languageList>
-          </configItem>
-        </variant>
-        <variant>
-          <configItem>
-            <name>tel-kagapa</name>
-            <!-- Keyboard indicator for Telugu layouts -->
-            <_shortDescription>te</_shortDescription>
-            <_description>Telugu (KaGaPa phonetic)</_description>
-            <languageList>
-              <iso639Id>tel</iso639Id>
-            </languageList>
-          </configItem>
-        </variant>
-        <variant>
-          <configItem>
-            <name>tel-sarala</name>
-            <!-- Keyboard indicator for Telugu layouts -->
-            <_shortDescription>te</_shortDescription>
-            <_description>Telugu (Sarala)</_description>
-            <languageList>
-              <iso639Id>tel</iso639Id>
-            </languageList>
-          </configItem>
-        </variant>
-        <variant>
-          <configItem>
-            <name>bolnagri</name>
-            <!-- Keyboard indicator for Hindi layouts -->
-            <_shortDescription>hi</_shortDescription>
-            <_description>Hindi (Bolnagri)</_description>
-            <languageList>
-              <iso639Id>hin</iso639Id>
-            </languageList>
-          </configItem>
-        </variant>
-        <variant>
-          <configItem>
-            <name>hin-wx</name>
-            <!-- Keyboard indicator for Hindi layouts -->
-            <_shortDescription>hi</_shortDescription>
-            <_description>Hindi (Wx)</_description>
-            <languageList>
-              <iso639Id>hin</iso639Id>
-            </languageList>
-          </configItem>
-        </variant>
-        <variant>
-          <configItem>
-            <name>hin-kagapa</name>
-            <!-- Keyboard indicator for Hindi layouts -->
-            <_shortDescription>hi</_shortDescription>
-            <_description>Hindi (KaGaPa phonetic)</_description>
-            <languageList>
-              <iso639Id>hin</iso639Id>
-            </languageList>
-          </configItem>
-        </variant>
-        <variant>
-          <configItem>
-            <name>san-kagapa</name>
-            <!-- Keyboard indicator for Sanskrit layouts -->
-            <_shortDescription>sa</_shortDescription>
-            <_description>Sanskrit (KaGaPa phonetic)</_description>
-            <languageList>
-              <iso639Id>san</iso639Id>
-            </languageList>
-          </configItem>
-        </variant>
-        <variant>
-          <configItem>
-            <name>mar-kagapa</name>
-            <!-- Keyboard indicator for Marathi layouts -->
-            <_shortDescription>mr</_shortDescription>
-            <_description>Marathi (KaGaPa phonetic)</_description>
-            <languageList>
-              <iso639Id>mar</iso639Id>
-            </languageList>
-          </configItem>
-        </variant>
-      </variantList>
-    </layout>
   </layoutList>
   <optionList>
       <group allowMultipleSelection="true">
diff --git a/rules/base.xml.in b/rules/base.xml.in
index e4e2cc5..a6f8f2a 100644
--- a/rules/base.xml.in
+++ b/rules/base.xml.in
@@ -1812,6 +1812,27 @@
     </layout>
     <layout>
       <configItem>
+        <name>bd</name>
+        <!-- Keyboard indicator for Bangla layouts -->
+        <_shortDescription>bn</_shortDescription>
+        <_description>Bangla</_description>
+        <languageList>
+          <iso639Id>ben</iso639Id>
+          <!-- sat-Beng: http://www.ethnologue.com/language/sat -->
+          <iso639Id>sat</iso639Id>
+        </languageList>
+      </configItem>
+      <variantList>
+        <variant>
+          <configItem>
+            <name>probhat</name>
+            <_description>Bangla (Probhat)</_description>
+          </configItem>
+        </variant>
+      </variantList>
+    </layout>
+    <layout>
+      <configItem>
         <name>in</name>
         <!-- Keyboard indicator for Indian layouts -->
         <_shortDescription>in</_shortDescription>
@@ -1820,6 +1841,286 @@
       <variantList>
         <variant>
           <configItem>
+            <name>ben</name>
+            <!-- Keyboard indicator for Bangla layouts -->
+            <_shortDescription>bn</_shortDescription>
+            <_description>Bangla (India)</_description>
+            <languageList>
+              <iso639Id>ben</iso639Id>
+              <!-- sat-Beng: http://www.ethnologue.com/language/sat -->
+              <iso639Id>sat</iso639Id>
+            </languageList>
+          </configItem>
+        </variant>
+        <variant>
+          <configItem>
+            <name>ben_probhat</name>
+            <!-- Keyboard indicator for Bangla layouts -->
+            <_shortDescription>bn</_shortDescription>
+            <_description>Bangla (India, Probhat)</_description>
+            <languageList>
+              <iso639Id>ben</iso639Id>
+              <!-- sat-Beng: http://www.ethnologue.com/language/sat -->
+              <iso639Id>sat</iso639Id>
+            </languageList>
+          </configItem>
+        </variant>
+        <variant>
+          <configItem>
+            <name>ben_baishakhi</name>
+            <_description>Bangla (India, Baishakhi)</_description>
+            <languageList>
+              <iso639Id>ben</iso639Id>
+              <!-- sat-Beng: http://www.ethnologue.com/language/sat -->
+              <iso639Id>sat</iso639Id>
+            </languageList>
+          </configItem>
+        </variant>
+        <variant>
+          <configItem>
+            <name>ben_bornona</name>
+            <_description>Bangla (India, Bornona)</_description>
+            <languageList>
+              <iso639Id>ben</iso639Id>
+              <!-- sat-Beng: http://www.ethnologue.com/language/sat -->
+              <iso639Id>sat</iso639Id>
+            </languageList>
+          </configItem>
+        </variant>
+         <variant>
+          <configItem>
+            <name>ben_gitanjali</name>
+            <_description>Bangla (India, Uni Gitanjali)</_description>
+            <languageList>
+              <iso639Id>ben</iso639Id>
+              <!-- sat-Beng: http://www.ethnologue.com/language/sat -->
+              <iso639Id>sat</iso639Id>
+            </languageList>
+          </configItem>
+        </variant>
+        <variant>
+          <configItem>
+            <name>ben_inscript</name>
+            <_description>Bangla (India, Baishakhi Inscript)</_description>
+            <languageList>
+              <iso639Id>ben</iso639Id>
+              <!-- sat-Beng: http://www.ethnologue.com/language/sat -->
+              <iso639Id>sat</iso639Id>
+            </languageList>
+          </configItem>
+        </variant>
+        <variant>
+          <configItem>
+            <name>eeyek</name>
+            <_description>Manipuri (Eeyek)</_description>
+            <languageList>
+              <iso639Id>mni</iso639Id>
+            </languageList>
+          </configItem>
+        </variant>
+        <variant>
+          <configItem>
+            <name>guj</name>
+            <!-- Keyboard indicator for Gujarati layouts -->
+            <_shortDescription>gu</_shortDescription>
+            <_description>Gujarati</_description>
+            <languageList>
+              <iso639Id>guj</iso639Id>
+            </languageList>
+          </configItem>
+        </variant>
+        <variant>
+          <configItem>
+            <name>guru</name>
+            <!-- Keyboard indicator for Punjabi layouts -->
+            <_shortDescription>pa</_shortDescription>
+            <_description>Punjabi (Gurmukhi)</_description>
+            <languageList>
+              <iso639Id>pan</iso639Id>
+            </languageList>
+          </configItem>
+        </variant>
+        <variant>
+          <configItem>
+            <name>jhelum</name>
+            <!-- Keyboard indicator for Punjabi layouts -->
+            <_shortDescription>pa</_shortDescription>
+            <_description>Punjabi (Gurmukhi Jhelum)</_description>
+            <languageList>
+              <iso639Id>pan</iso639Id>
+            </languageList>
+          </configItem>
+        </variant>
+        <variant>
+          <configItem>
+            <name>kan</name>
+            <!-- Keyboard indicator for Kannada layouts -->
+            <_shortDescription>kn</_shortDescription>
+            <_description>Kannada</_description>
+            <languageList>
+              <iso639Id>kan</iso639Id>
+            </languageList>
+          </configItem>
+        </variant>
+        <variant>
+          <configItem>
+            <name>kan-kagapa</name>
+            <!-- Keyboard indicator for Kannada layouts -->
+            <_shortDescription>kn</_shortDescription>
+            <_description>Kannada (KaGaPa phonetic)</_description>
+            <languageList>
+              <iso639Id>kan</iso639Id>
+            </languageList>
+          </configItem>
+        </variant>
+        <variant>
+          <configItem>
+            <name>mal</name>
+            <!-- Keyboard indicator for Malayalam layouts -->
+            <_shortDescription>ml</_shortDescription>
+            <_description>Malayalam</_description>
+            <languageList>
+              <iso639Id>mal</iso639Id>
+            </languageList>
+          </configItem>
+        </variant>
+        <variant>
+          <configItem>
+            <name>mal_lalitha</name>
+            <!-- Keyboard indicator for Malayalam layouts -->
+            <_shortDescription>ml</_shortDescription>
+            <_description>Malayalam (Lalitha)</_description>
+            <languageList>
+              <iso639Id>mal</iso639Id>
+            </languageList>
+          </configItem>
+        </variant>
+        <variant>
+          <configItem>
+            <name>mal_enhanced</name>
+            <!-- Keyboard indicator for Malayalam layouts -->
+            <_shortDescription>ml</_shortDescription>
+            <_description>Malayalam (enhanced Inscript, with rupee)</_description>
+            <languageList>
+              <iso639Id>mal</iso639Id>
+            </languageList>
+          </configItem>
+         </variant>
+         <variant>
+           <configItem>
+            <name>ori</name>
+            <!-- Keyboard indicator for Oriya layouts -->
+            <_shortDescription>or</_shortDescription>
+            <_description>Oriya</_description>
+            <languageList>
+              <iso639Id>ori</iso639Id>
+              <!-- sat-Orya: http://www.ethnologue.com/language/sat -->
+              <iso639Id>sat</iso639Id>
+            </languageList>
+          </configItem>
+        </variant>
+        <variant>
+           <configItem>
+            <name>olck</name>
+            <!-- Keyboard indicator for Ol Chiki layouts -->
+            <_shortDescription>sat</_shortDescription>
+            <_description>Ol Chiki</_description>
+            <languageList>
+              <!-- sat-Olck: http://www.ethnologue.com/language/sat -->
+              <iso639Id>sat</iso639Id>
+            </languageList>
+          </configItem>
+        </variant>
+        <variant>
+          <configItem>
+            <name>tam_unicode</name>
+            <!-- Keyboard indicator for Tamil layouts -->
+            <_shortDescription>ta</_shortDescription>
+            <_description>Tamil (Unicode)</_description>
+            <languageList>
+              <iso639Id>tam</iso639Id>
+            </languageList>
+          </configItem>
+        </variant>
+        <variant>
+          <configItem>
+            <name>tam_keyboard_with_numerals</name>
+            <!-- Keyboard indicator for Tamil layouts -->
+            <_shortDescription>ta</_shortDescription>
+            <_description>Tamil (keyboard with numerals)</_description>
+            <languageList>
+              <iso639Id>tam</iso639Id>
+            </languageList>
+          </configItem>
+        </variant>
+        <variant>
+          <configItem>
+            <name>tam_TAB</name>
+            <!-- Keyboard indicator for Tamil layouts -->
+            <_shortDescription>ta</_shortDescription>
+            <_description>Tamil (TAB typewriter)</_description>
+            <languageList>
+              <iso639Id>tam</iso639Id>
+            </languageList>
+          </configItem>
+        </variant>
+        <variant>
+          <configItem>
+            <name>tam_TSCII</name>
+            <!-- Keyboard indicator for Tamil layouts -->
+            <_shortDescription>ta</_shortDescription>
+            <_description>Tamil (TSCII typewriter)</_description>
+            <languageList>
+              <iso639Id>tam</iso639Id>
+            </languageList>
+          </configItem>
+        </variant>
+        <variant>
+          <configItem>
+            <name>tam</name>
+            <!-- Keyboard indicator for Tamil layouts -->
+            <_shortDescription>ta</_shortDescription>
+            <_description>Tamil</_description>
+            <languageList>
+              <iso639Id>tam</iso639Id>
+            </languageList>
+          </configItem>
+        </variant>
+        <variant>
+          <configItem>
+            <name>tel</name>
+            <!-- Keyboard indicator for Telugu layouts -->
+            <_shortDescription>te</_shortDescription>
+            <_description>Telugu</_description>
+            <languageList>
+              <iso639Id>tel</iso639Id>
+            </languageList>
+          </configItem>
+        </variant>
+        <variant>
+          <configItem>
+            <name>tel-kagapa</name>
+            <!-- Keyboard indicator for Telugu layouts -->
+            <_shortDescription>te</_shortDescription>
+            <_description>Telugu (KaGaPa phonetic)</_description>
+            <languageList>
+              <iso639Id>tel</iso639Id>
+            </languageList>
+          </configItem>
+        </variant>
+        <variant>
+          <configItem>
+            <name>tel-sarala</name>
+            <!-- Keyboard indicator for Telugu layouts -->
+            <_shortDescription>te</_shortDescription>
+            <_description>Telugu (Sarala)</_description>
+            <languageList>
+              <iso639Id>tel</iso639Id>
+            </languageList>
+          </configItem>
+        </variant>
+        <variant>
+          <configItem>
             <name>urd-phonetic</name>
             <!-- Keyboard indicator for Urdu layouts -->
             <_shortDescription>ur</_shortDescription>
@@ -1853,6 +2154,61 @@
         </variant>
         <variant>
           <configItem>
+            <name>bolnagri</name>
+            <!-- Keyboard indicator for Hindi layouts -->
+            <_shortDescription>hi</_shortDescription>
+            <_description>Hindi (Bolnagri)</_description>
+            <languageList>
+              <iso639Id>hin</iso639Id>
+            </languageList>
+          </configItem>
+        </variant>
+        <variant>
+          <configItem>
+            <name>hin-wx</name>
+            <!-- Keyboard indicator for Hindi layouts -->
+            <_shortDescription>hi</_shortDescription>
+            <_description>Hindi (Wx)</_description>
+            <languageList>
+              <iso639Id>hin</iso639Id>
+            </languageList>
+          </configItem>
+        </variant>
+        <variant>
+          <configItem>
+            <name>hin-kagapa</name>
+            <!-- Keyboard indicator for Hindi layouts -->
+            <_shortDescription>hi</_shortDescription>
+            <_description>Hindi (KaGaPa phonetic)</_description>
+            <languageList>
+              <iso639Id>hin</iso639Id>
+            </languageList>
+          </configItem>
+        </variant>
+        <variant>
+          <configItem>
+            <name>san-kagapa</name>
+            <!-- Keyboard indicator for Sanskrit layouts -->
+            <_shortDescription>sa</_shortDescription>
+            <_description>Sanskrit (KaGaPa phonetic)</_description>
+            <languageList>
+              <iso639Id>san</iso639Id>
+            </languageList>
+          </configItem>
+        </variant>
+        <variant>
+          <configItem>
+            <name>mar-kagapa</name>
+            <!-- Keyboard indicator for Marathi layouts -->
+            <_shortDescription>mr</_shortDescription>
+            <_description>Marathi (KaGaPa phonetic)</_description>
+            <languageList>
+              <iso639Id>mar</iso639Id>
+            </languageList>
+          </configItem>
+        </variant>
+        <variant>
+          <configItem>
             <name>eng</name>
             <!-- Keyboard indicator for English layouts -->
             <_shortDescription>en</_shortDescription>
-- 
cgit v0.10.2
From b3fb72308942f705e3e8e17765e4097773ed5483 Mon Sep 17 00:00:00 2001
From: Akshay S Dinesh <asdofindia@gmail.com>
Date: Mon, 19 Jun 2017 09:44:50 +1000
Subject: typofix tel-sarala key layout

This was causing error in loading new keyboard description for all
using Indian keyboards

Signed-off-by: Akshay S Dinesh <asdofindia@gmail.com>

diff --git a/symbols/in b/symbols/in
index 7658ee7..79d5099 100644
--- a/symbols/in
+++ b/symbols/in
@@ -1472,7 +1472,7 @@ xkb_symbols "tel-sarala"
     key <AC09> { [          U0C32,          U0C33,          U0C0C,          U0C61 ] }; // ల ళ ఌ ౡ
     key <AC10> { [          U0C1F,          U0C20,      semicolon,          colon ] }; // ట ఠ ; :
     key <AC11> { [     quoteright,       quotedbl	    	    		  ] }; // ' " 
-    key <AD01> { [          U0C46,          U0C0E,          U0C44           U0C34 ] }; // ె ఎ ౄ ఴ
+    key <AD01> { [          U0C46,          U0C0E,          U0C44,          U0C34 ] }; // ె ఎ ౄ ఴ
     key <AD02> { [          U0C38,          U0C37,          U0C44                 ] }; // స ష ౄ
     key <AD03> { [          U0C47,          U0C0F,          U0C44                 ] }; // ే ఏ ౄ
     key <AD04> { [          U0C30,          U0C31,          U0C44,          U0C60 ] }; // ర ఱ ౄ ౠ
-- 
cgit v0.10.2
 
